src/commands/help.ts

```diff
@@ -94,6 +95,11 @@ export class HelpCommand implements Command {
       {header: 'Command Options', optionList: command.args},
       {header: 'Global Options', optionList: globalArguments},
     ];
+
+    if (command.aliases.length) {
+      usageGroups.splice(1, 0, {header: 'Alias(es)', content: command.aliases});
```

Comment:
hard to tell from the PR, can you add to the PR description a sample of what this now looks like?